geowebcache/azureblob/src/test/java/org/geowebcache/azure/AzureBlobStoreConfigProviderTest.java

```diff
@@ -18,35 +18,10 @@
 import static org.junit.Assert.assertTrue;
 
 import com.thoughtworks.xstream.XStream;
-import org.geowebcache.GeoWebCacheExtensions;
-import org.junit.After;
-import org.junit.Before;
 import org.junit.Test;
-import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 public class AzureBlobStoreConfigProviderTest {
 
-    @Before
-    public void setUp() throws Exception {
```

Comment:
Please do not remove, the test is now to check that the environment variables can survive a xstream load (this is actually at the very core of the issue report), while there should be another test checking that variable expansion works.

Reply:
Reverted.